src/source/generator.scala

```diff
@@ -386,6 +386,33 @@ abstract class Generator(spec: Spec)
     w.w(end)
   }
 
+  def normalEnumOptions(e: Enum) = e.options.filter(_.specialFlag == None)
+
+  def writeEnumOptionNone(w: IndentWriter, e: Enum, ident: IdentConverter) {
+    for (o <- e.options.find(_.specialFlag == Some(Enum.SpecialFlag.None))) {
```

Comment:
Naming the special flag NoneFlag or NoFlags might be clearer. I was tripped up for a bit by the use of None above, vs. the use of Enum.SpecialFlag.None here.